BREAKING.md

```diff
@@ -122,6 +124,12 @@ The following interfaces used by the `PendingStateManager` are no longer exporte
 -   `IPendingState`
 -   `IPendingLocalState`
 
+### Remove deprecated ContainerServices in @fluidframework/aqueduct
+`ContainerServices` in `@fluidframework/aqueduct` was deprecated in 2.0.0-internal.3.0.0 and has now been removed.
+
+### Remove deprecated waitForAttach
+`waitForAttach()` was deprecated in 2.0.0-internal.3.0.0 and has now been removed.
+
```

Comment:
Consider instead of having these granular entries, just making it "Aqueduct members removed" (basically mirroring the "Aqueduct members deprecated" below).